quark/core/quark.py

```diff
@@ -49,6 +50,8 @@ def __init__(self, apk, core_library="androguard"):
         core_library = core_library.lower()
         if core_library == "rizin":
             self.apkinfo = RizinImp(apk)
+        elif core_library == "r2":
```

Comment:
I suggest adding "radare2" as a choice because it is the full name for r2.

Reply:
Done.